data_types/softlayer_virtual_guest.go

```diff
@@ -54,31 +54,29 @@ type SoftLayer_Virtual_Guest_Template_Parameters struct {
 	Parameters []SoftLayer_Virtual_Guest_Template `json:"parameters"`
 }
 
+
+//According the SL API Doc(http://sldn.softlayer.com/reference/services/SoftLayer_Virtual_Guest/createObject),
```

Comment:
Cool, you can remove this comment in code. The comment in the PR comment section is good enough